app/resources/views/pages/home.blade.php

```diff
@@ -1,6 +1,20 @@
 @extends('layouts.default')
-@section('content')
+@section('head-content')
+	<script>
+			var user_longitude = {{ $base_user->getLongitude() }};
+			var user_latitude  = {{ $base_user->getLatitude() }};
+			var address_default = "{{ $address_default }}";
```

Comment:
I used user_longitude and user_latitude to set the center of the map when the page loads. It is linked to the home.js